### PR TITLE
Wizard: Validate snapshot date before enabling the next button

### PR DESCRIPTION
**Pull request by croissanne**

### CODE REVIEW

**Comment by regexowl** [comment]
Why are the tests failing? Hmm... it can't find blueprint name... is it possible that the tests don't get past the snapshots step because Next is disabled?

**Comment by croissanne** [comment]
> Why are the tests failing? Hmm... it can't find blueprint name... is it possible that the tests don't get past the snapshots step because Next is disabled?

I'll take a look, I'll also disable the links on the left hand side in case the snapshot date is invalid.

**Comment by ezr-ondrej** [comment]
@croissanne could we tweak it to use https://github.com/osbuild/image-builder-frontend/blob/6e138b5274997f1b8fd21c06cef0bc1295e72ee8/src/Components/CreateImageWizard/utilities/useValidation.tsx ?

Something like https://github.com/osbuild/image-builder-frontend/pull/2281

If used, we also disable the Create button and we also have nice exclamation marks next to the step.

**Comment by ezr-ondrej** [comment]
It also uses a different strategy then disabling the links - you can traverse the form, but you've got the step marked as failed and can not create.

**Comment by ezr-ondrej** [comment]
Let me rephrase any objections agains me refactoring it once it's merged? :stuck_out_tongue:

**Comment by croissanne** [comment]
> @croissanne could we tweak it to use https://github.com/osbuild/image-builder-frontend/blob/6e138b5274997f1b8fd21c06cef0bc1295e72ee8/src/Components/CreateImageWizard/utilities/useValidation.tsx ?
> 
> Something like #2281
> 
> If used, we also disable the Create button and we also have nice exclamation marks next to the step.

Oh nice, sorry I had no idea this was going on. Yea I can close this and then wait until that PR

**Comment by ezr-ondrej** [comment]
I'm working on this in #2305
